src/libstore/globals.hh

```diff
@@ -826,8 +828,17 @@ public:
           > `.netrc`.
         )"};
 
-    /* Path to the SSL CA file used */
-    Path caFile;
+    Setting<Path> caFile{
+        this, getDefaultSSLCertFile(), "ssl-cert-file",
+        R"(
+          The path of a file containing CA certificates used to
+          authenticate `https://` downloads. It defaults to the first
+          of `/etc/ssl/certs/ca-certificates.crt` and
+          `/nix/var/nix/profiles/default/etc/ssl/certs/ca-bundle.crt`
+          that exists. It can be overriden using the
+          `NIX_SSL_CERT_FILE` and `SSL_CERT_FILE` environment variable
+          (in that order of precedence).
```

Comment:
```suggestion
          that exists.
          
          It can be overridden by environment variables, in order of precedence:
          1. `NIX_SSL_CERT_FILE`
          2. `SSL_CERT_FILE`
```

Both are currently undocumented. I wold like that added and linked to as part of this PR.
While there is no fancy information to be found there, it will make that setting more discoverable when we add a back-link, since we can reasonably expect people to skim the env var section. And it will make the env var section more discoverable as well.